sycl/include/sycl/kernel_bundle.hpp

```diff
@@ -183,9 +183,15 @@ class __SYCL_EXPORT kernel_bundle_plain {
 
   bool native_specialization_constant() const noexcept;
 
-  bool ext_oneapi_has_kernel(const std::string &name);
+  bool ext_oneapi_has_kernel(const std::string &name) {
+    return ext_oneapi_has_kernel(name.c_str());
+  }
+  bool ext_oneapi_has_kernel(detail::string_view name);
```

Comment:
As another note, these new `string_view` variants are not part of the extensions, so they should be made private. The free functions further down should be moved to the `detail` namespace.

Reply:
Yes, I moved the new method to the private section.